### PR TITLE
meta-lxatac-bsp: linux-lxatac: upgrade to 6.9

### PR DESCRIPTION
**Pull request by hnez**
umpf tag is now: 6.9/customers/lxa/lxatac/20240514-1
The tag includes the following topic branches:

  - v6.8/topic/reproducible-build
  - v6.7/topic/net-ksz-switch
  - v6.9/topic/stm32mp1-spi-fix-hang
  - v6.8/customers/lxa/lxatac

This means we got rid of two topic branches, thanks to Uwe's mainlining efforts, but gained one new topic branch:

  - v6.9/topic/stm32mp1-spi-fix-hang

    This topic branch reverts kernel commit 52b62e7a5d4fb
    ("spi: stm32: enable controller before asserting CS")
    which broke SPI on the LXA TAC.
    The issue has been reported upstream and the offending
    commit will likely be reverted upstream as well.

The series.inc file generated by umpf has again changed a bit and does not set the package PV anymore.
Since we have already migrated to using the series.inc-provided PV the linux-lxatac.bb was adapted to preserve the previous behaviour.

TODO before merging:

- [x] Since this is a PR into the new scarthgap branch, that does not yet contain scarthgap-y content #137 should be merged first, which performs the scarthgap update.

### CODE REVIEW

**Comment by hnez** [comment]
Hi, now that #137 is merged and the dust has settled (default branches are updated, build jobs are updated and ran successfully, labgrid tests are updated and also ran fine and the first scarthgap based [testing bundle](https://downloads.linux-automation.com/lxatac/software/testing/2024-05-30/) is available and installable) we can go ahead and merge this PR as well..

**Comment by hnez** [comment]
I've just noticed that this PR is still open. I think we should get this merged.

**Review by jluebbe — COMMENTED** [review]
LGTM from the Yocto side, but someone else should look at the kernel patches.

@hnez Is 0503-ARM-dts-stm32-lxa-tac-reduce-RGMII-interface-drive-s.patch no longer needed?

**Comment by hnez** [comment]
> @hnez Is 0503-ARM-dts-stm32-lxa-tac-reduce-RGMII-interface-drive-s.patch no longer needed?

I've had a look and it is actually no longer needed, since https://lore.kernel.org/all/20240304135612.814404-1-o.rempel@pengutronix.de/ was merged between 6.8 and 6.9.
That must have slipped through when writing the update commit message.

File: meta-lxatac-bsp/recipes-kernel/linux/files/defconfig [hunk 1]
```diff
@@ -5757,12 +5778,10 @@ CONFIG_FAT_DEFAULT_IOCHARSET="iso8859-1"
 # CONFIG_FAT_DEFAULT_UTF8 is not set
 CONFIG_EXFAT_FS=m
 CONFIG_EXFAT_DEFAULT_IOCHARSET="utf8"
-CONFIG_NTFS_FS=m
-# CONFIG_NTFS_DEBUG is not set
-# CONFIG_NTFS_RW is not set
 CONFIG_NTFS3_FS=m
 CONFIG_NTFS3_LZX_XPRESS=y
 # CONFIG_NTFS3_FS_POSIX_ACL is not set
+CONFIG_NTFS_FS=m
```
Comment (a3f) [inline meta-lxatac-bsp/recipes-kernel/linux/files/defconfig]: fyi, this can be dropped. it only serves to select `CONFIG_NTFS3_FS`.

**Review by a3f — APPROVED** [review]

File: meta-lxatac-bsp/recipes-kernel/linux/files/defconfig (hunk 1 above)
Comment (hnez) [reply meta-lxatac-bsp/recipes-kernel/linux/files/defconfig]: I've included this change and while at it also mentioned the RGMII drive strength change in the commit message.
Thanks for the review!

**Pull request merged**